image/golem_blender_app/golem_blender_app/commands/verify.py

```diff
@@ -104,16 +105,23 @@ def _collect_results(
         s[0] == SubtaskStatus.SUCCESS for s in subtasks_statuses.values()
     ])
     if not all_finished:
+        print('Not all finished, waiting for more results')
         return
 
+    print('All finished, collecting results')
     collector = RenderingTaskCollector(
         width=params['resolution'][0],
         height=params['resolution'][1],
     )
     for i in subtasks_nums[::-1]:
-        path = work_dir / f'subtask{subtasks_statuses[i][1]}' / 'results' / \
-            f'result{frame:04d}.{out_format}'
-        collector.add_img_file(str(path))
+        result_dir = work_dir / f'subtask{subtasks_statuses[i][1]}' / 'results'
+        result_img = result_dir / f'result{frame:04d}.{out_format}'
+        print(f'result_dir: {result_dir}')
+        for file in result_dir.iterdir():
```

Comment:
This name shadows the `file` is a builtin function

Reply:
good catch, will update